pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -484,3 +485,13 @@ func GetResourcesNamespace() string {
 	log.Errorf("There was an error fetching the namespace from the context, using default")
 	return "default"
 }
+
+// GetRESTObject allows to retrive a custom resource from the APIserver
+func (c *APIClient) GetRESTObject(path string, output runtime.Object) error {
+	result := c.Cl.CoreV1().RESTClient().Get().AbsPath(path).Do()
```

Comment:
You can actually do this in one line like

```
c.Cl.CoreV1().RESTClient().Get().AbsPath(path).Do().Into(result).Error()
```

If an error occurred `Into` will simply be a noop and then `Error` will return any error or `nil`.